dotnetcore2.0/run/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM microsoft/aspnetcore-build AS builder
+WORKDIR /source
+
+# caches restore result by copying csproj file separately
+COPY MockBootstraps/*.csproj .
+RUN dotnet restore
+
+# copies the rest of your code
+COPY MockBootstraps/ .
+RUN dotnet publish --output /app/ --configuration Release
+
+
+FROM lambci/lambda-base
+
+RUN rm -rf /var/runtime /var/lang && \
+  curl 'https://hm0s1q-db3pap001.files.1drv.com/y4mUBe_7LPKlEdvZRR8JKW44PuCG75m3w0N8eIHQy-SjfHdHlNCejlhjZE10BNL1aA-HT7I0-JDXB4w4XMQ5IHKHgT2oLCUjDw4itIcZBVb6VYEjdzVurzlRGgz0YsPOPnZMwQjxDX7EVShce6WXyMZSM6KblGfxg8FF62uZgtfd-2_hY2co3_OFVAhpQD2UaNrSh8GqESiaqEGrrnDxHJaQg/dotnet2.tgz?download&psid=1' | tar -zx -C /
+
+
+COPY --from=0 /app/MockBootstraps.* /var/runtime/
+
+ENTRYPOINT ["/var/lang/bin/dotnet", "/var/runtime/MockBootstraps.dll"]
```

Comment:
Just a question. So one example of how to invoke this would be 

```bash
/var/lang/bin/dotnet \
/var/runtime/MockBootstraps.dll \
AssemblyName::MyApp.Function::FunctionHandler \
'{"Records":[{"EventVersion":"1.0","EventSubscriptionArn":"arn:aws:sns:EXAMPLE","EventSource":"aws:sns","Sns":{"SignatureVersion":"1","Timestamp":"1970-01-01T00:00:00.000Z","Signature":"EXAMPLE","SigningCertUrl":"EXAMPLE","MessageId":"95df01b4-ee98-5cb9-9903-4c221d41eb5e","Message":"myMessage","MessageAttributes":{"Test":{"Type":"String","Value":"TestString"},"TestBinary":{"Type":"Binary","Value":"TestBinary"}},"Type":"Notification","UnsubscribeUrl":"EXAMPLE","TopicArn":"myTopic","Subject":"mySubject"}}]}'
```

Correct?

Reply:
Yes this is correct if you are inside the container.
If you use the container then you call it that way:
    docker run --rm -v $ASSEMBLY_DIR:/var/task containerName AssemblyName::MyAppClass.Function::FunctionHandler \
'{"Records":[{"EventVersion":"1.0","EventSubscriptionArn":"arn:aws:sns:EXAMPLE","EventSource":"aws:sns","Sns":{"SignatureVersion":"1","Timestamp":"1970-01-01T00:00:00.000Z","Signature":"EXAMPLE","SigningCertUrl":"EXAMPLE","MessageId":"95df01b4-ee98-5cb9-9903-4c221d41eb5e","Message":"myMessage","MessageAttributes":{"Test":{"Type":"String","Value":"TestString"},"TestBinary":{"Type":"Binary","Value":"TestBinary"}},"Type":"Notification","UnsubscribeUrl":"EXAMPLE","TopicArn":"myTopic","Subject":"mySubject"}}]}'

Reply:
@jason-riddle thx for the comments :+1: 
Unfortunately it was not intedned to add the dump project, but i will refactor it and add the changes to the PR